bedrock/entity/end_chat.py

```diff
@@ -0,0 +1,22 @@
+import asyncio
+import sys
+
+from temporalio.client import Client
+from workflows import EntityBedrockWorkflow
+
+
+async def main():
+    # Create client connected to server at the given address
+    client = await Client.connect("localhost:7233")
+
+    workflow_id = "entity-bedrock-workflow"
+
+    handle = client.get_workflow_handle(workflow_id)
```

Comment:
```suggestion
    handle = client.get_workflow_handle_for(EntityBedrockWorkflow.run, workflow_id)
```

Not required, but this is a bit more type-safe